core/src/apps/eos/sign_tx.py

```diff
@@ -0,0 +1,54 @@
+from trezor import wire
+from trezor.crypto.curve import secp256k1
+from trezor.crypto.hashlib import sha256
+from trezor.messages.EosSignedTx import EosSignedTx
+from trezor.messages.EosSignTx import EosSignTx
+from trezor.messages.EosTxActionRequest import EosTxActionRequest
+from trezor.messages.MessageType import EosTxActionAck
+from trezor.utils import HashWriter
+
+from apps.common import paths
+from apps.eos import writers
+from apps.eos.actions import process_action
+from apps.eos.helpers import validate_full_path
+from apps.eos.layout import require_sign_tx
+
+
+async def sign_tx(ctx, msg: EosSignTx, keychain):
+    if msg.chain_id is None:
+        raise wire.DataError("No chain id")
+    if msg.header is None:
+        raise wire.DataError("No header")
+    if msg.num_actions is None or msg.num_actions == 0:
+        raise wire.DataError("No actions")
+
+    await paths.validate_path(ctx, validate_full_path, path=msg.address_n)
```

Comment:
we have changed this again in the meantime. please update this call, see this commit https://github.com/trezor/trezor-firmware/commit/8aa60e6cfd84ba35613e5eed0088fb43e97aa8e0 for details

Reply:
Done